vignettes/immunedeconv.Rmd

```diff
@@ -124,26 +124,28 @@ In addition, human-based methods can be used to deconvolute mouse data through t
 to the corresponding human orthologues
 
 ```R
-gene_expression_matrix <- immunedeconv::mouse_genes_to_human(gene_expression_matrix)
+gene_expression_matrix <- immunedeconv::convert_human_mouse_genes(gene_expression_matrix, convert_to = "human")
```

Comment:
Did I get this right that the function wasn't renamed, but just the wrong function reference from the tutorial?

Reply:
The function was renamed because instead of going from mouse genes to human, now it can go both ways

Reply:
This is also a breaking change then... Usually what is done in such cases is to keep the old function as an alias to the new one (provided that it has a compatible signature) and show a deprecation warning when it is still used.